lib/data-types.js

```diff
@@ -350,6 +393,22 @@ DATEONLY.prototype._stringify = function _stringify(date) {
   return moment(date).format('YYYY-MM-DD');
 };
 
+DATEONLY.prototype._sanitize = function _sanitize(value, options) {
+  if (!options || options && !options.raw) {
+    return moment(value).format('YYYY-MM-DD');
```

Comment:
I think this is missing a `!!value` check. Now, we are getting `"Invalid date"` value for `NULL`, resulting in an error.